components/transfer/commons/transfer_references.lua

```diff
@@ -235,6 +238,14 @@ function TransferRef.createReference(refData, date)
 				name = referenceKey
 			}
 		}
+	elseif refType == TOURNAMENT_LEAVE_TYPE then
```

Comment:
if i see it correctly on phone this if branch is the same as the one above, just zhe type is different
kick this if branch and change line 233 from
`elseif refType == TOURNAMENT_TYPE then` to
`elseif refType == TOURNAMENT_TYPE or refType == TOURNAMENT_LEAVE_TYPE then`

possibly check if we can combine further ones